buildroot-external/patches/occu/0127-WebUI-Sort-FirmwareOverview.patch

```diff
@@ -0,0 +1,18 @@
+--- occu/WebUI/www/webui/webui.js.orig
++++ occu/WebUI/www/webui/webui.js
+@@ -20568,6 +20568,15 @@
+               }
+               conInfo("Check firmware version of all devices - number of devices: " + self.devList.length);
+               if (self.devList.length > 0) {
++								function sortorder_devList(a , b){
```

Comment:
Bitte hier keine Tabs, sondern Spaces verwenden. Auch bitte immer sich am jeweiligen Space/Tab Style der Datei orientieren und auch die Anzahl an Einrückungen exakt gleich machen wie das im rest der Datei so ist.

Reply:
Sorry, ist geändert

Reply:
Nein, ist nicht geändert in diesem PR.

Reply:
Okay, also man muss nach dem Commit auch noch Push drücken 🙈